georgian/README.md

```diff
@@ -0,0 +1,9 @@
+# Georgian
+
+There is no official standard for Georgian braille. However a group of
+very dedicated people from the Georgian Braille printing house and the
+Georgian School For Blind maintain a de-facto standard.
+
+- Origins and history of [Georgian Braille](Georgian Braille.docx)
```

Comment:
This .docx only contains a couple of simple paragraphs, so I suggest to just put it here within the README:

> Georgian Braille was adopted in 1890s. In 1892 the first boarding-school for blind was opened in Tbilisi. Georgian blind students and teachers used Georgian Braille only in handwriting. The Braille printed books were supplied from Russia since late 1960s. In 1967 a printing house Skhivi was established in Tbilisi Georgia. The printing house was owned by the Georgian Blind Union. 
> 
> In 1990s, after the collapse of Soviet Union, the printing press stopped working until 2005. The new generation Braille books were introduced to blind users in 2005 and school textbooks are being printed since 2012. There is still only one printing press in Georgia. The Braille printing is fully based on the letters and symbols which were used in 1967-1992. 
>
> The Georgian Braille is standardized, due to the fact that there is only one printing house and the persons who are employed there are skilled in use of Braille. They are the teachers of the school for blind and members of the Georgian Blind Union. There is only one magazine printed in Braille in Georgia and that is also printed in the same printing house. The national exams to enter the university are prepared by the personnel employed at the Braille printing house
>
> With additional questions regarding the Georgian Braille, interested persons can address:
Mariam Mikiashvili deputy director of the School For Blind at
Mariam.miki@gmail.com
>
> Khatuna Gogidze director of the Braille printing house at 
gogidzekhatuna@gmail.com

Reply:
Yes, good idea. Done